src/components/SingleAnswer.tsx

```
@@ -0,0 +1,62 @@
+            key={`radio-${item}`}
+            value={item}
+            style={{ marginBottom: "0.5rem" }}
+            label={<Text typo="body1">{item}</Text>}
+          />
+        ))}
+        <RadioButton
+          key={`radio-기타`}
+          value="기타"
+          inputProps={{
+            onClick: () => {
+              textInputRef.current && textInputRef.current.focus();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)